test/streamer.js

```diff
@@ -49,7 +49,7 @@ describe('streaming', () => {
     return streamer('https://streamer.test/streams/data.log', ws)
       .then(() => expect(ws.data.join(''), 'to equal', 'My retried data'))
       .then(() => api.done())
-  })
+  }).timeout(6000)
```

Comment:
I think this and the other would be more readable as `5 * timeout * 1.2`. That's easy to understand as five attempts plus some execution overhead.

Reply:
Done.